website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -545,13 +570,26 @@ var QuickSearchNodeDisplay = {
             return m('.', args.eligibleNodes().slice(0, args.countDisplayed()).map(function(n){
                 var project = args.nodes()[n];
                 var numContributors = project.embeds.contributors.links.meta.total;
+                var title = project.attributes.title;
+                try {
+                    var root = project.embeds.root.data.attributes.title;
+                }
+                catch (err) {
+                    var root = "Private Project";
```

Comment:
You *should* be able to get the error details in here to determine if the root is private or not. If not, obviously that's more complicated and we may have to skip it. But most likely you'll have `project.embeds.root.error` or similar that you can check.